.github/workflows/eks-ci.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           key: ${{ runner.os }}-dockers-cache-${{ env.CONTAINER_SUFFIX }}
 
       - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1-node16
+        uses: aws-actions/configure-aws-credentials@v1.7.0
```

Comment:
@oren-zohar, we cannot bump version due to [this issue](https://github.com/aws-actions/configure-aws-credentials/issues/632). It will return us again to node12.